meson.build

```diff
@@ -2,7 +2,7 @@ project('lite-xl',
     ['c'],
     version : '2.1.0',
     license : 'MIT',
-    meson_version : '>= 0.47',
+    meson_version : '>= 0.60',
```

Comment:
I'd prefer not to bump the meson version if the suggested workaround works (https://github.com/lite-xl/lite-xl/pull/997#discussion_r882284373).